starter-core/src/test/groovy/io/micronaut/starter/feature/graalvm/GraalNativeImageSpec.groovy

```diff
@@ -137,9 +126,9 @@ class GraalNativeImageSpec extends BeanContextSpec implements CommandOutputFixtu
     void 'deploy.sh script is not created for an default app with feature graalvm for language: #language'(Language language) {
         when:
         def output = generate(
-                ApplicationType.DEFAULT,
-                new Options(language, TestFramework.JUNIT, BuildTool.MAVEN),
-                ['graalvm']
+            ApplicationType.DEFAULT,
+            new Options(language, TestFramework.JUNIT, BuildTool.MAVEN, JdkVersion.JDK_11),
```

Comment:
11 is the default. why did you set explicitely Jdk 11 ? Just to be convey that it uses 11?